antlr4ide.ui/src/main/java/com/github/jknack/antlr4ide/ui/preferences/GeneralPreferencePage.java

```diff
@@ -0,0 +1,54 @@
+package com.github.jknack.antlr4ide.ui.preferences;
+
+import com.github.jknack.antlr4ide.console.ConsoleImpl;
+import org.eclipse.core.runtime.preferences.InstanceScope;
+import org.eclipse.jface.preference.ComboFieldEditor;
+import org.eclipse.jface.preference.FieldEditorPreferencePage;
+import org.eclipse.jface.preference.IPreferenceStore;
+import org.eclipse.ui.IWorkbench;
+import org.eclipse.ui.IWorkbenchPreferencePage;
+import org.eclipse.ui.preferences.ScopedPreferenceStore;
+import org.eclipse.xtend.expression.LoggingNullEvaluationHandler.Level;
```

Comment:
Use 'org.apache.logging.log4j.Level' instead and reduce xtext dependencies or does xtext add value here wrt logging?